modules/installation-configure-proxy.adoc

```diff
@@ -225,6 +226,7 @@ Operator then creates a `trusted-ca-bundle` config map that merges the contents
 with the {op-system} trust bundle. The `additionalTrustBundle` field is required unless
 the proxy's identity certificate is signed by an authority from the {op-system} trust
 bundle.
+<5> Optional: The policy to determine the configuration of the `Proxy` object to reference the `user-ca-bundle` config map in the `trustedCA` field. The allowed values are `Proxyonly` and `Always`. Use `Proxyonly` to reference the `user-ca-bundle` config map only when `http/https` proxy is configured, and `Always` to always reference the `user-ca-bundle` config map. The default value is `Proxyonly`.
```

Comment:
```suggestion
<5> Optional: The policy to determine the configuration of the `Proxy` object to reference the `user-ca-bundle` config map in the `trustedCA` field. The allowed values are `Proxyonly` and `Always`.  The default value is `Proxyonly`. Use `Proxyonly` to reference the `user-ca-bundle` config map only when `http/https` proxy is configured. Use `Always` to always reference the `user-ca-bundle` config map.
```
This is just a minor suggestion. You have done a good job using the principles of minimalism here. I think that by breaking up the long sentence, using parallel structure, and moving the `Proxyonly` information closer together, it is a little bit easier to read. WDYT?